.github/workflows/build.yml

```diff
@@ -39,5 +39,9 @@ jobs:
       - name: "Upload artifacts 📤"
         uses: actions/upload-artifact@v4
         with:
-          name: webpack-bundle
-          path: dist/
+          path: |
+            dist/
+            lib/
+
+      - name: "Run pack dry-run 📦"
+        run: yarn pack --filename conda-store-ui.tgz --dry-run
```

Comment:
I was thinking... should we upload the tarball as the artifact? Listing out the paths in the upload-artifact action is a bit dangerous because the real source of truth for what paths should go into the tarball is `package.json` > "files"

Reply:
Good shout. I am refactoring this because I built off what was already in the repo (albeit minimal) but I am not really happy with it.